api/server/router/swarm/backend.go

```diff
@@ -14,8 +16,8 @@ type Backend interface {
 	Update(uint64, types.Spec) error
 	GetServices(basictypes.ServiceListOptions) ([]types.Service, error)
 	GetService(string) (types.Service, error)
-	CreateService(types.ServiceSpec) (string, error)
-	UpdateService(string, uint64, types.ServiceSpec) error
+	CreateService(types.ServiceSpec, http.Header) (string, error)
```

Comment:
Alright, I can do that. At this point we have the resolved token, so I'll just pass that through.